puppet/modules/web/manifests/vhost/downloads.pp

```diff
@@ -36,8 +48,8 @@
 
   file { "${downloads_directory}/HEADER.html":
     ensure => file,
-    owner  => 'root',
-    group  => 'root',
+    owner  => $user,
+    group  => $user,
```

Comment:
I think it's still fine to keep this owned by root. This signals that it's managed by Puppet, rather than the user.

Reply:
agreed